src/event-input/calcPb.js

```diff
@@ -13,26 +16,25 @@ const moment = require('moment');
  * @returns { { result: object, runner: object } } The modified result and runner objects
  */
 const calcPb = (result, runner) => {
-    let newPb = false;
+    let newPb = 0;
     
-    let distance = result.distance;
+    let distance = result.distance.toString();
     if (!['2', '5'].includes(distance)) throw new Error('distance invalid');
 
     let fastest = runner.stats[`records${distance}k`].fastest;
     let slowest = runner.stats[`records${distance}k`].slowest;
 
 
-    if (!fastest) {
+    if (!fastest /* Therefore also !slowest */) {
```

Comment:
This comment confuses me. I think you should place it on the next line. I'm guessing you're saying:
```
// There isn't a fastest time, thus there is no slowest time, so set them both
```